src/Resources/ShipmentPiece.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Mvdnbrk\DhlParcel\Resources;
+
+class ShipmentPiece extends Piece
+{
```

Comment:
I think this can lead to confusion, when someone looks in the code he might think label_id is required to create a piece. But maybe that is just me :)

Reply:
In my opinion it will be more cumbersome to create separate classes for what what we are sending to the API and what we are receiving back. Both classes will share the same properties and/or functionality.